src/temp.py

```diff
@@ -0,0 +1,60 @@
+'''
+file: temp.py
+author: Carson Berry <carsonberry@hotmail.ca>
+Date: January 30th, 2021
+
+Purpose: To read data from the one-wire interface of the DS18B20 temperature sensor on teh raspberry pi. 
+This file is designed to be used in a obj oriented way, such as temp.read()
+
+inputs: null
+outputs: float temp
+
+Usage:
+import temp
+python temp.read()
+'''
+
+import os 
+import time
+
+
+
+#name of specific temperature sensor in given system. Should try and automate this process. 
+#temp_sensor = '/sys/bus/w1/devices/28-3c01b556d3de/w1_slave'
```

Comment:
Maybe we include a setup to a better device name? Is that what you mean?

Reply:
Yeah, this is what I meant, maybe we can use regex - I think this temp sensor always starts with 28-*